### ISSUE TITLE
Give every nation +1 colonist at X diplo tech to encourage everyone to get in on the colony game without having to invest in an NI. 

### ISSUE THREAD

**Issue by Dragonkn1ght** [RohanTLK/Make-Europa-Great-Again#28]

**Comment by ComradeL1berty** [comment]
Good idea, you could make it a decision that checks for countries without explo/expansion so colonisers dont get an extra colonist for no reason.

**Comment by SpectreOfRohan** [comment]
Will need to increase the colonial range that is given at a specific diplo tech as it's kinda hard for nations to colonise without taking the +100000% colonial range NI

**Comment by Dragonkn1ght** [comment]
This needs to be sufficiently late as to not trivialise the early mover advantage colonials get (1520-30?). Can include a colonial range bonus in the decision i guess.

**Comment by SpectreOfRohan** [comment]
So what I've done is left it until diplo tech 11 but I've also not increased colonial range much (in the decision) so that most of the more inland nations (like venice and west of lubeck) will need to make a deal to secure islands or something. The colonial range increase should be JUST enough for most nations to be able to core a hopping island.
